lib/gradient.ex

```diff
@@ -31,20 +31,27 @@ defmodule Gradient do
   @spec type_check_file(String.t(), options()) :: :ok | :error
   def type_check_file(path, opts \\ []) do
     opts = Keyword.put(opts, :return_errors, true)
+    module = Keyword.get(opts, :module, "all_modules")
 
-    with {:ok, forms} <- ElixirFileUtils.get_forms(path),
-         {:elixir, _} <- wrap_language_name(forms) do
-      forms = maybe_specify_forms(forms, opts)
-
-      case maybe_gradient_check(forms, opts) ++ maybe_gradualizer_check(forms, opts) do
-        [] ->
-          :ok
-
-        errors ->
-          opts = Keyword.put(opts, :forms, forms)
-          ElixirFmt.print_errors(errors, opts)
-          :error
-      end
+    with {:ok, forms} <- ElixirFileUtils.get_forms(path, module),
```

Comment:
```suggestion
    with {:ok, asts} <- ElixirFileUtils.get_forms(path, module),
```

The count of the entities in this list, therefore the structure of this piece of data, would become clearer if instead of calling forms from a single module `forms`, we called it `ast` (Abstract Syntax Tree). Then the list of ASTs would be `asts`, instead of "a list of lists of forms".

Reply:
That makes sens, changed to `asts` and `ast`